pmm-app/src/pmm-qan/panel/components/Details/Metadata/Metadata.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Overlay } from 'shared/components/Elements/Overlay/Overlay';
+import { Collapse, useStyles2 } from '@grafana/ui';
+import { Table } from '../../../../../shared/components/Elements/Table';
+import { LinkTooltip } from '../../../../../shared/components/Elements/LinkTooltip/LinkTooltip';
+import { getStyles } from './Metadata.styles';
+import { metadataTableHead } from './Metadata.constants';
+
+const Metadata = ({ metadata, loading }) => {
+  const styles = useStyles2(getStyles);
+  const [isMetadataOpen, setMetadataVisibility] = useState(true);
```

Comment:
nit: usually it is called `[<name>, set<name>]`, for example [metadataOpen, setMetadataOpen]